.gitlab-ci.yml

```diff
@@ -506,7 +506,7 @@ test:acceptance:
   script:
     # Upload files: first .buildinfo and .deb, then .changes
     - for d in output/opensource/*; do
-    -   find "$d" -name '*.buildinfo' -o -name '*.deb' -o -name '*.orig.tar.*' -o -name '*.debian.tar.*' -o -name '*.dsc' | while read -r file; do
+    -   find "$d" \( -name '*.buildinfo' -o -name '*.deb' -o -name '*.orig.tar.*' -o -name '*.debian.tar.*' -o -name '*.dsc' \) -and -not -path "${d}/orig/*" | while read -r file; do
```

Comment:
By design, this is still wrong. The publish job should not need to filter `/output/opensource/orig/` because these are not artifacts of any of the `build:packages:<distro>:<version>:<arch>` jobs, they are artifacts from ``build:orig:debian:buster:amd64` which the publish job does not depend on.

At the high level, this is what should happen:
* `build:orig:debian:buster:amd64 ` produces `output/opensource/orig/*.orig.tar.gz`
* `build:packages:<distro>:<version>:amd64` produces `/output/opensource/<distro>-<version>-amd64/*.orig.tar.gz` but does **not** propagate `output/opensource/orig/*.orig.tar.gz`.

Does it make sense? For example let's take `mender-artifact` files from the build in this PR:
```
./job-build-debian-arm64/output/opensource/orig/mender-artifact_3.10.0~git20220926.4a5b4a0.orig.tar.gz
./job-build-debian-arm64/output/opensource/debian-bullseye-arm64/mender-artifact_3.10.0~git20220926.4a5b4a0-1+debian+bullseye+builder652559327_arm64.deb
./job-build-debian-arm64/output/opensource/debian-bullseye-arm64/mender-artifact_3.10.0~git20220926.4a5b4a0-1+debian+bullseye+builder652559327_arm64.buildinfo
./job-build-debian-arm64/output/opensource/debian-bullseye-arm64/mender-artifact_3.10.0~git20220926.4a5b4a0-1+debian+bullseye+builder652559327_arm64.changes
./job-build-debian-arm64/output/opensource/debian-bullseye-arm64/mender-artifact-deb-version
./job-build-debian-amd64/output/opensource/orig/mender-artifact_3.10.0~git20220926.4a5b4a0.orig.tar.gz
./job-build-debian-amd64/output/opensource/debian-bullseye-amd64/mender-artifact_3.10.0~git20220926.4a5b4a0-1+debian+bullseye+builder652559327.debian.tar.xz
./job-build-debian-amd64/output/opensource/debian-bullseye-amd64/mender-artifact_3.10.0~git20220926.4a5b4a0.orig.tar.gz
./job-build-debian-amd64/output/opensource/debian-bullseye-amd64/mender-artifact_3.10.0~git20220926.4a5b4a0-1+debian+bullseye+builder652559327.dsc
./job-build-debian-amd64/output/opensource/debian-bullseye-amd64/mender-artifact_3.10.0~git20220926.4a5b4a0-1+debian+bullseye+builder652559327_amd64.changes
./job-build-debian-amd64/output/opensource/debian-bullseye-amd64/mender-artifact_3.10.0~git20220926.4a5b4a0-1+debian+bullseye+builder652559327_amd64.buildinfo
./job-build-debian-amd64/output/opensource/debian-bullseye-amd64/mender-artifact-deb-version
./job-build-debian-amd64/output/opensource/debian-bullseye-amd64/mender-artifact_3.10.0~git20220926.4a5b4a0-1+debian+bullseye+builder652559327_amd64.deb
```

There `/output/opensource/orig/mender-artifact_3.10.0~git20220926.4a5b4a0.orig.tar.gz` is duplicated (and gets overridden in the publish job`) but they should just not be propagated.

I think this can be achieved by selectively `mv` and not `cp -v ... *` [here](https://github.com/mendersoftware/mender-dist-packages/pull/281/files#diff-a696dd766c3d4eb6a58ba77a8c75d4f3888cf19cb5d7332f12a6214f07f0d5d1R124). The script knows which package is building, so it should be able to `mv` only that one and leave the rest for successive runs.

Reply:
I agree. how about now? (I took slightly different path)